dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -224,4 +224,134 @@ public void testFolderWithFileCopy()
+                    .setLength(length).build();
+    WriteOptions writeOptions =
+            WriteOptions.newBuilder().setOverwrite(false).setCheckContent(true).build();
+    UfsReadOptions read =
+            UfsReadOptions.newBuilder().setUser("test").setTag("1").setPositionShort(false).build();
+    ListenableFuture<List<RouteFailure>> move =
+            mWorker.move(Collections.singletonList(route), read, writeOptions);
+    List<RouteFailure> failures = move.get();
+    Assert.assertEquals(0, failures.size());
+    Assert.assertTrue(b.exists());
+    Assert.assertFalse(a.exists());
+    try (InputStream in = Files.newInputStream(b.toPath())) {
+      byte[] readBuffer = new byte[length];
+      while (in.read(readBuffer) != -1) {
+      }
+      Assert.assertArrayEquals(buffer, readBuffer);
+    }
+  }
+
+  @Test
+  public void testMoveException() throws IOException, ExecutionException, InterruptedException {
+    File srcRoot = mTestFolder.newFolder("srcException");
+    File dstRoot = mTestFolder.newFolder("dstException");
+    // create test file under mSrcFolder
+    File a = new File(srcRoot, "a");
+    a.createNewFile();
+    File b = new File(dstRoot, "b");
+    int length = 10;
+    byte[] buffer = BufferUtils.getIncreasingByteArray(length);
+    BufferUtils.writeBufferToFile(a.getAbsolutePath(), buffer);
+    Route route =
+            Route.newBuilder().setDst(b.getAbsolutePath()).setSrc(a.getAbsolutePath())
+                    .setLength(length).build();
+    WriteOptions writeOptions = WriteOptions.newBuilder().setOverwrite(false).build();
+    ListenableFuture<List<RouteFailure>> move =
+            mWorker.move(Collections.singletonList(route), null, writeOptions);
+    List<RouteFailure> failures = move.get();
+    Assert.assertEquals(1, failures.size());
+    Assert.assertFalse(b.exists());
+  }
+
+  @Test
+  public void testSingleFolderMove() throws IOException, ExecutionException, InterruptedException {
+    File srcRoot = mTestFolder.newFolder("src");
+    File dstRoot = mTestFolder.newFolder("dst");
+    // create test file under mSrcFolder
+    File a = new File(srcRoot, "a");
+    a.mkdirs();
+    File b = new File(dstRoot, "b");
+    Route route =
+            Route.newBuilder().setDst(b.getAbsolutePath()).setSrc(a.getAbsolutePath()).build();
+    WriteOptions writeOptions =
+            WriteOptions.newBuilder().setOverwrite(false).setCheckContent(true).build();
+    UfsReadOptions read =
+            UfsReadOptions.newBuilder().setUser("test").setTag("1").setPositionShort(false).build();
+    ListenableFuture<List<RouteFailure>> move =
+            mWorker.move(Collections.singletonList(route), read, writeOptions);
+    List<RouteFailure> failures = move.get();
+    Assert.assertEquals(0, failures.size());
+    Assert.assertTrue(b.exists());
+    Assert.assertTrue(b.isDirectory());
+    Assert.assertFalse(a.exists());
+  }
+
+  @Test
+  @Ignore
```

Comment:
I didn't realize this is ignored when I'm reviewing the PR and could you help fix this test since it's broken